src/tracker/face_id/face_net/inception_resnet_v1.py

```diff
@@ -358,12 +358,3 @@ def get_torch_home():
 
 #     state_dict = torch.load(cached_file)
 #     mdl.load_state_dict(state_dict)
-
-
-def get_torch_home():
```

Comment:
It's okay to delete this because there is a byte-for-byte identical one defined on line 324 that this one was shadowing. My guess is that you tried to cut-and-paste to move it elsewhere, but accidentally copied and pasted instead.